syntax = "proto3";

message Payment {
    int32 id = 1;
    int32 user_id = 2;
    int32 order_id = 3;
    float amount = 4;
    string status = 5;
    string created_at = 6;
    string updated_at = 7;
}

message PaymentCreate {
    int32 user_id = 1;
    int32 order_id = 2;
    float amount = 3;
    string status = 4;
}

message PaymentUpdate {
    int32 user_id = 1;
    int32 order_id = 2;
    float amount = 3;
    string status = 4;
}
